src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java

```diff
@@ -329,38 +339,55 @@ public static ItemStack harvestPlant(@Nonnull Block block) {
         }
 
         for (Berry berry : getBerries()) {
-            if (item.getId().equalsIgnoreCase(berry.getID())) {
+            if (item.getID().equalsIgnoreCase(berry.getID())) {
```

Comment:
`getID()` is deprecated, use `getId()`
```suggestion
            if (item.getId().equals(berry.getId())) {
```

Reply:
nO I like it better and I'm a rebel deal with it